### ISSUE TITLE
TestDriven.Net tries to run tests in .NET Core projects that don't contain tests

### ISSUE THREAD

**Issue by jcansdale** [jcansdale/TestDriven.Net-Issues#23]

**Comment by mwhelan** [comment]
That rang a bell for me but can't find it now. However, I do get a similar(?) issue. When I right click solution and Run Tests, TD.Net runs all the test projects successfully but it also tries to run tests against a non-test project and gives this error message:

```
------ Test started: Assembly: Specify.Examples.dll ------

Couldn't find an appropriate testRunner to execute your tests.
Please ensure your project is a Console Application and has one of the following as a dependency:
"dotnet-test-nunit"
"dotnet-test-xunit"

Alternatively, any method or property without a test attribute can be executed as an "Ad hoc" test.
```

Here is the project.json for the non-test project:

```
{
  "version": "1.0.0-*",

  "dependencies": {
    "NETStandard.Library": "1.6.0"
  },

  "frameworks": {
    "netstandard1.6": {
      "imports": "dnxcore50"
    }
  }
}
```

**Comment by jcansdale** [comment]
@mwhelan Thanks for the report. I think this is to do with the different way TestDriven.Net detects test projects in .xproj projects compared to the classic .NET Framework. I've changed the subject of this issue to track it.

P.S. I've started preemptively creating GitHub issues to be a landing page for things that might go wrong (they're linked from error messages). That way if they become a problem, there is somewhere obvious to find a solution or report the problem.
